### DISCUSSION TITLE
Fork Tracing

### DISCUSSION THREAD

**Discussion by ysk3a** [stackblitz/core#2451] [Ideas]
A feature under the information section of the project that shows, inplace/another route/another tab/somewhere, which stackblitz project the forked from by project id or something.
Just like githubs `Insights > Forks`.
Would be nice to be able to see who forked from who and who in a tree of forks is the root/original.

e.g. case private/not found project root:
[private original project shows page as private] > forked project with 'forked from private project/deleted project] > ...

e.g case private/not found project not root:
original public project > forked from original. this project was public but changed to private > forked from a once public but now private project but here is the next closest public project of the project (in this case original public project) > etc...

e.g. all public/not found
easy tree link to parent project else not found next closest parent project

i guess this sounds complex but at least some info that the project was forked from somewhere else. then later one linking to be able to see the original project.

**Comment by UltraBlackLinux** [comment]
I'm in the need for that as well.
